src/drivers/barometer/bmp388/bmp388.cpp

```diff
@@ -45,13 +45,16 @@
 BMP388::BMP388(IBMP388 *interface, const char *path) :
 	CDev(path),
 	ScheduledWorkItem(px4::device_bus_to_wq(interface->get_device_id())),
-	_px4_baro(interface->get_device_id()),
+	_px4_baro(interface->get_device_id(), ORB_PRIO_MAX),
```

Comment:
Why ORB_PRIO_MAX? You might want to choose a slightly lower priority that still achieves what you want/need. Then if someone optionally connects a higher quality external barometer it can become the primary.

Reply:
Good catch, I'll move to ORB_PRIO_DEFAULT